extensions/blocks/instagram-gallery/image-transition.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState, useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+export default function ImageTransition( { src } ) {
+	const [ loaded, setLoaded ] = useState( false );
+	const [ size, setSize ] = useState( 'auto' );
```

Comment:
It's probably because I still remember the previous version of this component, but by reading "size" I was thinking that this would have affected both height and width, of both placeholder and image.

What about making it super self-explanatory?
```js
const [ containerHeight, setContainerHeight ] = useState( 'auto' );
```

Reply:
This makes sense - done.